### ISSUE TITLE
Evaluation mode ignored for dropout list

### ISSUE THREAD

**Issue by JanMLuts** [namisan/mt-dnn#45]
Dear,

while a nn.ModuleList() is used for the scoring_list, this is not the case for the dropout_list:

https://github.com/namisan/mt-dnn/blob/eb0aef4cbcdbde097f4535580af64ebaf9666108/mt_dnn/matcher.py#L16-L24

This seems to have an impact when switching the network mode in class MTDNNModel from training to evaluation. The scoring_list appears to be correctly switched to evaluation mode while this is not happening for the dropout_list. As a consequence, dropout is active at prediction time.

Jan Luts

**Comment by namisan** [comment]
Thanks Jan. I just fixed this issue.

**Comment by JanMLuts** [comment]
No worries Xiaodong. Great research!

Jan
